web/tellus_app/tellus/settings.py

```diff
@@ -345,3 +346,8 @@ def get_database_key():
 
     },
 }
+
+DATAPUNT_AUTHZ = {
+    'JWT_SECRET_KEY': os.getenv('JWT_SHARED_SECRET_KEY', 'ecuresniinsecure'),
```

Comment:
... dan hoef je hier geen default waarde mee te geven, en dan kan die ook nooit per ongeluk in productie terecht komen 🥇  Je moet dan ook even een testkey zetten in `/docker-compose.yml`.

Het is niet veiliger want de tokens worden hier alleen geverifieerd, niet gemaakt. Maar het is wel netter: de middleware vertelt je expliciet als hij geen goede key heeft, maar als hij de default key heeft kan hij alleen maar zeggen dat de signature niet klopt.